ldes-server-infra-postgres/postgres-admin-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/postgres/eventstream/EventStreamPostgresRepository.java

```diff
@@ -1,47 +1,59 @@
-import be.vlaanderen.informatievlaanderen.ldes.server.admin.postgres.eventstream.repository.EventStreamEntityRepository;
 import be.vlaanderen.informatievlaanderen.ldes.server.admin.postgres.eventstream.service.EventStreamConverter;
+import be.vlaanderen.informatievlaanderen.ldes.server.admin.postgres.eventstream.v2.mapper.EventStreamMapper;
+import be.vlaanderen.informatievlaanderen.ldes.server.admin.postgres.eventstream.v2.repository.EventStreamEntityRepository;
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.EventStream;
-import org.springframework.stereotype.Component;
+import com.google.common.collect.Streams;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Repository;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.util.List;
 import java.util.Optional;
 
-@Component
+@Repository
 public class EventStreamPostgresRepository implements EventStreamRepository {
+	private final be.vlaanderen.informatievlaanderen.ldes.server.admin.postgres.eventstream.repository.EventStreamEntityRepository oldModelRepo;
 	private final EventStreamEntityRepository repository;
 	private final EventStreamConverter converter = new EventStreamConverter();
 
-	public EventStreamPostgresRepository(EventStreamEntityRepository repository) {
-		this.repository = repository;
+	public EventStreamPostgresRepository(be.vlaanderen.informatievlaanderen.ldes.server.admin.postgres.eventstream.repository.EventStreamEntityRepository oldModelRepo, @Qualifier("collectionsRepository") EventStreamEntityRepository repository) {
+		this.oldModelRepo = oldModelRepo;
+        this.repository = repository;
 	}
 
 	@Override
 	public List<EventStream> retrieveAllEventStreams() {
-		return repository.findAll()
+		final var oldStreams = oldModelRepo.findAll()
 				.stream()
-				.map(converter::toEventStream)
-				.toList();
+				.map(converter::toEventStream);
+		final var newStreams = repository.findAll().stream()
+				.map(EventStreamMapper::fromEntity);
+		return Streams.concat(oldStreams, newStreams).distinct().toList();
 	}
 
 	@Override
 	public Optional<EventStream> retrieveEventStream(String collectionName) {
-		return repository.findById(collectionName).map(converter::toEventStream);
+		return repository
+				.findByName(collectionName)
+				.map(EventStreamMapper::fromEntity)
+				.or(() -> oldModelRepo.findById(collectionName).map(converter::toEventStream));
 	}
 
 	@Override
 	@Transactional
 	public EventStream saveEventStream(EventStream eventStream) {
-		repository.save(converter.fromEventStream(eventStream));
+		oldModelRepo.save(converter.fromEventStream(eventStream));
+		repository.save(EventStreamMapper.toEntity(eventStream));
 		return eventStream;
 	}
 
 	@Override
 	@Transactional
 	public void deleteEventStream(String collectionName) {
-		repository.deleteById(collectionName);
+		repository.deleteByName(collectionName);
+		oldModelRepo.deleteById(collectionName);
```

Comment:
As mentioned in the estimation, how you solve the story is up to you, but at the end of the story, I would drop the "oldModelRepo". So i don't expect this in the final PR :)